### DISCUSSION TITLE
New checkmarks not as clear as I would like them to be

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1286] [Ideas]
The new default checkmarks in the treeview to the left are, while beautiful, not really readable at a glance; the checkmark and the minus are too small and the box around them isn't helping things either.

A simple green checkmark and a red diagonal cross, would (imho) be way more helpful. The box around the symbol provides additional clutter which needs to be interpreter by the user.

Example of what I mean attached below. I know, my artistic skills are absolutely mind boggling. :)
![cross](https://user-images.githubusercontent.com/85426904/206908909-a5b769f1-8076-4550-bb35-298145eaf25d.png)
![checkmark](https://user-images.githubusercontent.com/85426904/206908911-823be650-82fa-47fb-b3ee-529e37a939bf.png)

**Comment by vkbo** [comment]
So, you basically want them reverted to the previous symbols?

**Reply by vkbo** [reply]
For the record, the checked checkbox is the icon provided by the icon theme novelWriter is using, so it is designed to be consistent with the other icons. I made the cross and minus ones to match the one provided by the designer.

The old icons were the OK and Cancel icons, and some users disliked that the same icons were used for different things on the UI, hence the switch to the "correct" icons from the theme.

**Reply by xahodo** [reply]
Well, to be honest, I don't mind icons being used in multiple places, as long as it's clear what they mean. In this case the icon is not really clear and can't instantly be interpreted by the user. imho it needs some love.

I've just figured out how to change things, so I'll go make my own changes. :)

**Reply by vkbo** [reply]
There was a long discussion on this already with multiple people engaging in the discussion, and this was the outcome. It is not a matter of lack of attention, rather one of preference.

I have an alternate set of icons I designed where the checked checkbox is the same, the current crossed one is an unchecked checkbox instead (this is the pair of icons from the actual icon theme) and a filled in box replacing the minus box. These are still an alternative. I will not revert back to the old icons. It is hard to accommodate opposing opinions, so that one will stand for now.
